app/src/main/java/com/fstyle/structure_android/screen/main/MainActivity.java

```diff
@@ -12,15 +14,21 @@
 import com.fstyle.structure_android.R;
 import com.fstyle.structure_android.data.model.User;
 import com.fstyle.structure_android.data.repository.UserRepository;
-import com.fstyle.structure_android.data.source.remote_api.error.BaseException;
-import com.fstyle.structure_android.data.source.remote_api.service.NameServiceClient;
+import com.fstyle.structure_android.data.source.local.UserLocalDataSource;
+import com.fstyle.structure_android.data.source.remote.UserRemoteDataSource;
+import com.fstyle.structure_android.data.source.remote.config.error.BaseException;
+import com.fstyle.structure_android.data.source.remote.config.service.NameServiceClient;
```

Comment:
ERROR: Unused import: com.fstyle.structure_android.data.source.remote.config.service.NameServiceClient. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports